ci/package_list.sh

```diff
@@ -8,12 +8,20 @@ RELEASE_PACKAGES=(
     tedge-watchdog
     tedge-apt-plugin
     tedge-log-plugin
-    c8y-configuration-plugin
+    tedge-configuration-plugin
     c8y-remote-access-plugin
     c8y-firmware-plugin
 )
 export RELEASE_PACKAGES
 
+# Deprecated packages are still built but not explicitly tested
+# This allows users to still access the packages if needed, however
+# it is only reserved for packages with a more public facing API
+DEPRECATED_PACKAGES=(
+    c8y-configuration-plugin
```

Comment:
I opted against this because the log plugin never had a publicly documented public interface (which what I was trying to allude to in the comment above the array definition.